<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DzielenieWidget</class>
 <widget class="QWidget" name="DzielenieWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DzielenieWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>X</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSpinBox" name="XSpin">
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="XSlider">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="XDial">
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Y</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSpinBox" name="YSpin">
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="YSlider">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="YDial">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>x/y=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dzielenieValue">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Setting for x and y values</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>minimum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="minimumValue">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>34</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>maximum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="maximumValue">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>XSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
